barretenberg/cpp/src/barretenberg/proof_system/relations/relation_types.hpp

```diff
@@ -34,49 +48,72 @@ template <typename Relation, size_t subrelation_index> constexpr bool subrelatio
     }
 }
 
+/**
+ * @brief Compute the full subrelation lenghhs, i.e., the lengths when regarding the challenges as
+ * variables.
+ */
+template <typename RelationImpl>
+consteval std::array<size_t, RelationImpl::SUBRELATION_PARTIAL_LENGTHS.size()> compute_full_subrelation_lengths()
+{
```

Comment:
if this is regarding challenges as variables, shouldn't this be total length or am I misunderstanding something?

Reply:
Yeah I started with "full" and then decided "total" was better but I missed the renaming here, thanks.